es2-persistence/src/main/java/com/netflix/conductor/dao/es/index/ElasticSearchDAO.java

```diff
@@ -206,11 +206,11 @@ private void initIndex() throws Exception {
 			}
 		}
 
-		//2. Mapping for the workflow document type
-		GetMappingsResponse response = elasticSearchClient.admin().indices().prepareGetMappings(indexName).addTypes(WORKFLOW_DOC_TYPE).execute().actionGet();
-		if (response.mappings().isEmpty()) {
+		//2. Add Mappings for the workflow document type
+		GetMappingsResponse getMappingsResponse = elasticSearchClient.admin().indices().prepareGetMappings(indexName).addTypes(WORKFLOW_DOC_TYPE).execute().actionGet();
+		if (getMappingsResponse.mappings().isEmpty()) {
 			logger.info("Adding the workflow type mappings");
-			InputStream stream = ElasticSearchDAO.class.getResourceAsStream("/wfe_type.json");
```

Comment:
same here, not needed anymore? remove the file if yes.

Reply:
Yes, files are removed.